style.css

```diff
@@ -1,5 +1,1007 @@
+    color: #172b4d;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+  }
+
+  textarea {
+    background: #fff;
+    width: 447px;
+    height: 170px;
+    padding-left: 20px;
+    padding-top: 10px;
+    margin-top: 10px;
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+  }
+
+  .highlight-4 {
+    background: #fff;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid white;
+    width: 150px;
+    margin-top: 15px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .font-6 {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .highlight-4:hover {
+    background: #6070ff;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid #6070ff;
+    width: 150px;
+    margin-top: 15px;
+  }
+
+  .font-6:hover {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: white;
+  }
+
+  .highlight-4:visited {
+    background: #2230d2;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid #2230d2;
+    width: 150px;
+    margin-top: 15px;
+  }
+
+  .font-6:visited {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: white;
+  }
+
+  .highlight-4:disabled {
+    background: #e5e5e5;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid #5e6c84;
+    width: 150px;
+    margin-top: 15px;
+  }
+
+  .font-6:disabled {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #5e6c84;
+  }
+
+  .shape {
+    display: none;
+    position: relative;
+    width: 134px;
+    height: 5px;
+    left: calc(50% - 134px / 2 - 0.5px);
+    bottom: 2px;
+    background: #0d151e;
+    border-radius: 10px;
+  }
+}
+
 @media (max-width: 375px) {
   body {
     background-color: white;
```

Comment:
- [ ] Great job so far with your project 👍 but setting this media query here breaks your page between the widths 376px and 991px. It would be great if you could address this issue. I've included a screenshot of your first page for reference but note that your whole site breaks and not just the first page.
- Your design:
![c1](https://user-images.githubusercontent.com/85113567/149405195-b126b398-277b-4c89-b3a1-26791f1e2871.jpg)

Reply:
the rest changes I will fix them but I have a problem with this when I put the mobile version code outside the media query it got mixed with the desktop version and the site doesn't work so please can you give me any help about that?